.github/workflows/test.yml

```diff
@@ -202,6 +202,12 @@ jobs:
            COVERALLS=1 CONTAINER_RUNNABLE=1 make check
         popd
 
+    - name: Check custom OVN image build works
```

Comment:
I guess we can remove this block now, right?

Reply:
I'm not sure whether that's very interesting though.  OVN is always backwards compatible from a schema perspective.  And I don't think there are other breakages that can be determined at build time.

Reply:
OK, you're right, that is useful indeed.